Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -574,13 +632,31 @@ public void OnLoggedOut()
             userLoggedIn = false;
         }
 
+        /// <summary>
+        /// Unsubscribes from invitations and Peer2Peer message requests.
+        /// This should be called before the application finishes shutting down in order to properly remove Native notifications.
+        /// TODO: This function has no references, and is incomplete in its implementation. This will likely be removed.
+        /// </summary>
+        [Obsolete]
```

Comment:
Neat! I didn't know this was something that could be done. Thanks for teaching me something new! 😀

Reply:
Handy tag is handy! ^o^ You can also use it to point users towards the new function that should be called, in the argument for this attribute.